panel/dist/css/widgetbox.css

```diff
@@ -4,6 +4,4 @@
   border-radius: 4px;
   -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
   box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
-  overflow-x: hidden;
-  overflow-y: hidden;
```

Comment:
These were hiding the calendar display. I can't think of a scenario to crop the widget display, so I removed this.